File: app/src/main/java/com/daviddf/geeklabtest/Loop2.java
package com.daviddf.geeklabtest;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

public class Loop2 extends AppCompatActivity {
    private final int SPLASH_DISPLAY_LENGTH = 10;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_loop2);
        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent mainIntent = new Intent(Loop2.this, Consentimiento.class);
                Loop2.this.startActivity(mainIntent);
                Loop2.this.finish();
            }
        }, SPLASH_DISPLAY_LENGTH);
    }
}

File: app/src/main/java/com/daviddf/geeklab/loop/Loop2.kt
package com.daviddf.geeklab.loop

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import android.os.Looper
import androidx.appcompat.app.AppCompatActivity
import com.daviddf.geeklab.R

class Loop2 : AppCompatActivity() {
    private val SPLASH_DISPLAY_LENGTH = 10

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_loop2)
        Handler(Looper.getMainLooper()).postDelayed({
            val mainIntent = Intent(this@Loop2, Consentimiento::class.java)
            startActivity(mainIntent)
            finish()
        }, SPLASH_DISPLAY_LENGTH.toLong())
    }
}
